{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/main.js

```diff
@@ -47,4 +48,7 @@ document.addEventListener('DOMContentLoaded', () => {
 
   // Remove pre loading class, used for fixing some filer/animation issues on load
   removePreloadClass()
+
+  // Initialise front end editing
+  setupEditing()
```

Comment:
This should be wrapped in a `require.ensure` - there's no reason non-admin users of the site should have to load all of the editing code.